lib/src/DOtherSide.cpp

```diff
@@ -174,6 +177,32 @@ void dos_qguiapplication_initialize_opengl()
     QGuiApplication::setAttribute(Qt::AA_ShareOpenGLContexts);
 }
 
+// This catches the QT and QML logs and outputs them.
+// This is necessary on Windows, because otherwise we do not get any logs at all
+void myMessageOutput(QtMsgType type, const QMessageLogContext &context, const QString &msg)
+{
+    QByteArray localMsg = msg.toLocal8Bit();
+    const char *file = context.file ? context.file : "";
+    const char *function = context.function ? context.function : "";
+    switch (type) {
+    case QtDebugMsg:
+        fprintf(stderr, "Debug: %s (%s:%u, %s)\n", localMsg.constData(), file, context.line, function);
```

Comment:
```suggestion
        fprintf(stdout, "Debug: %s (%s:%u, %s)\n", localMsg.constData(), file, context.line, function);
```

Reply:
stdout didn't work. I'll keep stderr for now 🤷